### DISCUSSION TITLE
Is strimzi-kafka support debian image

### DISCUSSION THREAD

**Discussion by chanvoyage** [strimzi/strimzi-kafka-operator#5354] [Q&A]
hello  everyone

i have some question about base images of strimzi-kafka , Currently Strimzi use redhat base images as picture  Is possible to change form redhat to debian  To be worried  about  problem my be occur

if i use redhat images which version Can i use  ? 

thank you very much 

<img width="469" alt="Screen Shot 2564-07-23 at 11 11 48" src="https://user-images.githubusercontent.com/39899497/126736778-af0be4ca-e11b-4769-aa85-4a4c5158adbd.png">

**Comment by scholzj** [comment]
> i have some question about base images of strimzi-kafka , Currently Strimzi use redhat base images as picture Is possible to change form redhat to debian To be worried about problem my be occur

You can maintain your own set of Dockerfiles in parallel directory structure: https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/DEV_GUIDE.md#alternative-docker-base-image ... keep in mind that Debian and RHEL based distributions are very different. Not sure if it affects the scripts in any way etc. You would need to try it.

> if i use redhat images which version Can i use ?

Sorry, I'm not sure I follow the question. You can use the images / Dockerfiles as they are. Th UBI8 images are free to use and redistribute. You do not need to have any Red Hat subscription or anything.

**Reply by chanvoyage** [reply]
thank you very much
